google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageOptions.java

```diff
@@ -652,9 +665,15 @@ public StorageRetryStrategy getStorageRetryStrategy() {
       return StorageRetryStrategy.getDefaultStorageRetryStrategy();
     }
 
+    /** This method is obsolete. Use {@link #getTerminationAwaitDurationDuration()} instead. */
+    @ObsoleteApi("Use getTerminationAwaitDurationDuration() instead")
+    public org.threeten.bp.Duration getTerminationAwaitDuration() {
+      return toThreetenDuration(getTerminationAwaitDurationDuration());
+    }
+
     /** @since 2.14.0 */
-    public Duration getTerminationAwaitDuration() {
-      return Duration.ofMinutes(1);
+    public java.time.Duration getTerminationAwaitDurationDuration() {
+      return java.time.Duration.ofMinutes(1);
```

Comment:
I don't think we've seen this case before (or at least I don't recall). But if the method ends with Duration then we come across `DurationDuration` and I think that's even more confusing.

Maybe we want to keep this for consistency with all other SDK methods, but I'm thinking that naming scheme isn't ideal.

Perhaps `getTerminationAwaitJavaTimeDuration()`? Open to suggestions for this.

Reply:
I agree with the point in https://github.com/googleapis/java-datastore/pull/1671#discussion_r1854745739 saying that using `DurationJavaTime` keeps the prefixes consistent

Reply:
to confirm this with @sydney-munro and @JesseLovelace